### DISCUSSION TITLE
Digital Watch

### DISCUSSION THREAD

**Discussion by DSalvigni** [grafana/grafana#37698] [Dashboards, Panels & Visualizations]
Hello Folk,
is it possible to have a digital watch as official plugin to be shown in the dashboards. It would be nice one shot into the panel the GMT and UTC time and eventually local time too.
The following plugin stopped to work: https://grafana.com/grafana/plugins/grafana-clock-panel/?tab=installation

From version 8.x.x

**Comment by zuchka** [comment]
>  The following plugin stopped to work: https://grafana.com/grafana/plugins/grafana-clock-panel/?tab=installation

This plugin still works for me on 8.1.  It's not working for you under any circumstances?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
